poll/public/handlebars/poll_results.handlebars

```diff
@@ -0,0 +1,43 @@
+<script id="poll-results-template" type="text/html">
+    <h3 class="poll-header">{{display_name}}</h3>
+    <div class="poll-question-container">{{{question}}}</div>
+    <ul class="poll-answers-results poll-results">
+    {{#each tally}}
+        <li class="poll-result">
+            <div class="poll-result-input-container">
+              <input id="answer-{{key}}" type="radio" disabled {{#if choice}}checked="True"{{/if}} />
+            </div>
+            {{#if any_img}}
+                <div class="poll-image result-image">
+                    <label for="answer-{{key}}" class="poll-image-label">
+                        {{#if img}}
+                          <img src="{{img}}" />
+                        {{/if}}
+                    </label>
+                </div>
+            {{/if}}
+            <div class="percentage-gauge-container">
+              <div class="percentage-gauge" style="width:{{percent}}%;">
+                  <label class="poll-answer-label" for="answer-{{key}}">{{{answer}}}</label>
+              </div>
+            </div>
+            <div class="poll-percent-container">
+                <span class="poll-percent-display{{#if first}} poll-top-choice{{/if}}">{{percent}}%</span>
+            </div>
+        </li>
+       {{^last}}
+        <li class="poll-spacer">
```

Comment:
@smarnach It's actually pretty bad for screen reader users.  I just played around with the CSS and found that you can use ```border-bottom: 4px solid transparent;```  on the ```<li>``` 's to achieve the same affect you are getting with the blank ```<li>```